app/controllers/publishers_controller.rb

```diff
@@ -64,9 +62,30 @@ def create_done
 
   def update
     publisher = current_publisher
+    update_params = publisher_update_params
+
+    current_email = publisher.email
```

Comment:
minor: my intuition tells me this conditional email confirmation flow should be extracted into a service class

Reply:
It did become complex enough that I can see that a service would be warranted 👍